packages/venia-concept/src/ServiceWorker/registerRoutes.js

```diff
@@ -17,34 +21,36 @@ import { THIRTY_DAYS, MAX_NUM_OF_IMAGES_TO_CACHE } from './defaults';
  * @returns {void}
  */
 export default function() {
-    const catalogCacheHandler = createCatalogCacheHandler();
+    const imageCacheHandler = createImageCacheHandler();
 
     registerRoute(
         new RegExp('(robots.txt|favicon.ico|manifest.json)'),
         new StaleWhileRevalidate()
     );
 
     /**
-     * Route that checks for resized catalog images in cache.
+     * Route that checks for resized images in cache.
      */
-    registerRoute(isResizedCatalogImage, ({ url, request, event }) => {
+    registerRoute(isResizedImage, ({ url, request, event }) => {
         const sameOrLargerImagePromise = findSameOrLargerImage(url, request);
         event.waitUntil(sameOrLargerImagePromise);
         return sameOrLargerImagePromise.then(
-            response =>
-                response || catalogCacheHandler.handle({ request, event })
+            response => response || imageCacheHandler.handle({ request, event })
         );
     });
 
     /**
      * Route to handle all types of images. Stores them in cache with a
      * cache name "images". They auto expire after 30 days and only 60
      * can be stored at a time.
+     *
+     * There is another route that handles images without width and options on them.
+     * This route handles images that wont have width options on them.
```

Comment:
I'm a little confused by this comment. This route handles images that won't have width options on them, or that will?